README.md

```diff
@@ -78,6 +82,12 @@ be any production-ready Pod providers. Only the one prototypes in multiple place
 - [solid.community](https://solid.community/) - Hosted by (?)) on (?)).
 - [Solid Web](https://solidweb.org/) - Hosted by Matthias Evering on (?)
 
+## Servers
+
+- [PHP Solid Server](https://github.com/pdsinterop/solid-php-server)
```

Comment:
If it's "PHP Solid Server" then why not call the repo "php-solid-server" instead of "solid-php-server"?

Reply:
Good point! Fixed.